Frontend/src/app/feed/feed.component.ts

```diff
@@ -3,6 +3,8 @@ import { DefaultService } from "../default.service";
 import { environment } from 'src/environments/environment';
 import { ToastrService } from 'ngx-toastr';
 import { DomSanitizer } from '@angular/platform-browser';
+import { AuthenticationService } from '../services/authentication.service';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
I am getting error that we are unable to import module

Reply:
Try executing `npm install --force`